RELEASES.md

```diff
@@ -5,6 +5,10 @@
 ### Fixes
 - `taffy::compute_layout` has been made public allowing Taffy to be used with custom storage (#263)
 
+### Fixes
+
```

Comment:
We now have a duplicate heading here

Reply:
Good spot, I missed that during the rebase.